VM Application Healthchecking

adiprerepa opened the pull request:
Implementation of the application healthchecking for Virtual Machines.

Design Doc (@howardjohn): https://docs.google.com/document/d/1G4s_PrmvkwCKoH_kXVeSUAszsaj3ZfpIlraSdeTSPKU/edit#
xds-proxy author: @rshriram

new PR because apparently, I cant use git

adiprerepa committed:
initial commit

adiprerepa committed:
initial commit

istio-testing commented:
@adiprerepa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-notes_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3024) | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-notes_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3024) | `/test release-notes_istio`
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

shamsher31 commented:
Adding ref to your old PR https://github.com/istio/istio/pull/26997 because review comments will be useful to understand what changes have been made.

@adiprerepa Anyways what problem you are facing with git. Let me know if you need any help?

adiprerepa committed:
Add xds healthcheck call

adiprerepa committed:
Add xds healthcheck call

adiprerepa commented:
@shamsher31 thanks for the ref. i screwed up a rebase, which brought in a bunch of commits, so i figured that opening a new PR would be easier. I think I am good now. The other one was more of a draft/wip anyway, this is *more* complete.

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-notes_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3024) | `/test release-notes_istio`
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio | 6abcb0cbfbae93ddfb6ae8a4038ba08690468fad | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19495) | `/test lint_istio`
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-pilot-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-pilot-k8s-tests_istio/19776) | `/test integ-pilot-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-pilot-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-pilot-k8s-tests_istio/19776) | `/test integ-pilot-k8s-tests_istio`
integ-telemetry-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-telemetry-k8s-tests_istio/19553) | `/test integ-telemetry-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-notes_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3026) | `/test release-notes_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-pilot-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-pilot-k8s-tests_istio/19776) | `/test integ-pilot-k8s-tests_istio`
integ-telemetry-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-telemetry-k8s-tests_istio/19553) | `/test integ-telemetry-k8s-tests_istio`
integ-multicluster-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10226) | `/test integ-multicluster-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adiprerepa committed:
lint

adiprerepa committed:
lint

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
lint_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19497) | `/test lint_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-pilot-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-pilot-k8s-tests_istio/19776) | `/test integ-pilot-k8s-tests_istio`
integ-telemetry-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-telemetry-k8s-tests_istio/19553) | `/test integ-telemetry-k8s-tests_istio`
integ-multicluster-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10226) | `/test integ-multicluster-k8s-tests_istio`
release-notes_istio | 2f5c410cd31b09663f9729d5fb4b9a1b10617b6e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3029) | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
unit-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/unit-tests_istio/20261) | `/test unit-tests_istio`
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-operator-controller-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-operator-controller-tests_istio/7930) | `/test integ-operator-controller-tests_istio`
integ-distroless-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-distroless-k8s-tests_istio/12802) | `/test integ-distroless-k8s-tests_istio`
gencheck_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/gencheck_istio/18085) | `/test gencheck_istio`
integ-ipv6-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-ipv6-k8s-tests_istio/5774) | `/test integ-ipv6-k8s-tests_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-pilot-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-pilot-k8s-tests_istio/19776) | `/test integ-pilot-k8s-tests_istio`
integ-telemetry-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-telemetry-k8s-tests_istio/19553) | `/test integ-telemetry-k8s-tests_istio`
integ-multicluster-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10226) | `/test integ-multicluster-k8s-tests_istio`
release-notes_istio | 2f5c410cd31b09663f9729d5fb4b9a1b10617b6e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3029) | `/test release-notes_istio`
lint_istio | 2f5c410cd31b09663f9729d5fb4b9a1b10617b6e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19500) | `/test lint_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adiprerepa committed:
new type of lint

adiprerepa committed:
new type of lint

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-test_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-test_istio/19366) | `/test release-test_istio`
integ-security-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-security-k8s-tests_istio/19884) | `/test integ-security-k8s-tests_istio`
integ-telemetry-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-telemetry-k8s-tests_istio/19553) | `/test integ-telemetry-k8s-tests_istio`
integ-multicluster-k8s-tests_istio | e38a56a2145d5b1c55e1097173b88e21d25bff45 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10226) | `/test integ-multicluster-k8s-tests_istio`
lint_istio | 2f5c410cd31b09663f9729d5fb4b9a1b10617b6e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/lint_istio/19500) | `/test lint_istio`
release-notes_istio | 4f86e075e2399d5281b786fe196b3fc798cde14f | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3030) | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

stevenctl committed:
fix echo tcp service (#26835)

* fix echo tcp service

* restore default timeout

* better logging in echo

* add failing unit test for echo tcp

* test response fields are present

* more logging

* revert timeout change

* expand test

* try to cause flake

* more useful log

* concurrency limit

* remove ut

* remove max concurrency

* run sniffing test

* read bytes with eof

* format

* spelling

* remove extraneous logs

* dont log buffer contents

* check equals EOF

* remove PROXY_XDS_VIA_AGENT

* remove extraneous changes

* temp disable proxy xds via agent

* override retry options

* disable cluster check for mirror test

* use multierror group to count errors

* bump max concurrency

* format

* enable all sniffing tests in single cluster mode

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-notes_istio | 4f86e075e2399d5281b786fe196b3fc798cde14f | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3030) | `/test release-notes_istio`
integ-multicluster-k8s-tests_istio | 4f86e075e2399d5281b786fe196b3fc798cde14f | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10230) | `/test integ-multicluster-k8s-tests_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

stevenctl committed:
use cp topology as default cfg topology (#27162)

* use cp topology as default cfg topology

* add logs

* lint

linsun committed:
Use precheck instead of verify-install (#27152)

* Adding logs for operator init

* fix lint

* suggest users to use precheck instead

ramaraochavali committed:
refactor push context (#27161)

* split push context in simple sub structs

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* move to pointer

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* move away from pointer

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* lint

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* unnecessary changes

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

rshriram committed:
Enable Istio Agent DNS interception for demo and preview (#27069)

* Enable Istio Agent DNS interception for demo and preview

* undo

* release notes

* fix

Signed-off-by: Shriram Rajagopalan <rshriram@tetrate.io>

* yaml fix

Signed-off-by: Shriram Rajagopalan <rshriram@tetrate.io>

* Update demo.yaml

* Update preview.yaml

* Update main.go

* Update iop-integration-test-defaults.yaml

istio-testing committed:
Automator: update istio/api@master dependency in istio/istio@master (#27170)

hacktron95 committed:
add deprecated message, attach it to deprecated commands (#26917)

* add deprecated message, attach it to deprecated commands

* add func to deprecate commands with a suffix

* avoid assignOp error by using += operator

* use table driven test for command-deprecate function

* add space before deprecated message, fix typo

stevenctl committed:
re-enable grpc and skip tcp traffic sniffing tests (#27171)

* re-enable grpc and skip tcp traffic sniffing tests

* fix typo

istio-testing committed:
Automator: update proxy@master in istio/istio@master (#27173)

howardjohn committed:
Remove custom readiness logic from test framework (#27178)

* This is broken if there are any VirtualServices changing the xds
config (such as weighted)
* This is broken if there is any Sidecar limiting scope
* This should not be needed. We have a readiness probe to handle
readiness. If it does not work, it indicates a bug in our code which is
something users will experience

stevenctl committed:
disable tcp again (#27176)

istio-testing committed:
Automator: update proxy@master in istio/istio@master (#27182)

howardjohn committed:
Wait for pod in TestAddToAndRemoveFromMesh (#27192)

\#27178 removed some waiting logic. This adds it back, due to test
flakes:
https://prow.istio.io/view/gs/istio-prow/logs/integ-pilot-k8s-tests_istio_postsubmit/3647

howardjohn committed:
Remove access log from preview (#27191)

I think this was accidentally added

devincd committed:
Fix spelling mistakes (#27197)

linsun committed:
small typo (#27190)

nmittler committed:
Fixing cert makefiles to support docs (#27186)

The Makefiles have recently been changed so that the generated files for self-signed certs have a `selfSigned` prefix. This requires extra steps to rename the files before generating the `cacerts` secret, which assumes certain file names.

This breaks the Makefile into 2: one for self-signed certs and another for certs based on a root CA from an existing k8s cluster.

nmittler committed:
Fix for includes in tools/certs Makefiles (#27221)

bianpengyuan committed:
Trim job suffix to extract out cron job name for workload metadata (#27195)

* special case cron job processing in webhook

* update

ostromart committed:
Output progress to console, capture log, create archive, cleanup (#27025)

* Output progress to console, capture log, create archive, cleanup

* Fix binary params

nmittler committed:
Preserve cert-chain.pem when generating self-signed certs (#27224)

howardjohn committed:
tf: update readme and delete empty file (#27227)

istio-testing committed:
Automator: update proxy@master in istio/istio@master (#27229)

howardjohn committed:
Fix crash when first DNS requests comes before NDS is loaded (#27228)

linsun committed:
create help for make files (#27225)

ramaraochavali committed:
streamline xds proxy logs (#27199)

* streamline xds proxy logs

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* remove extra space

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

howardjohn committed:
nds: add debug endpoint (#27230)

I did not make this a part of config_dump since that is mirror envoy
format and will be sent over the wire for istioctl (maybe?) and we don't
currently have a wrapper proto like all the other types

Commit:
Fix null ingressGateways for minimal and empty profile (#27198)

Commit:
Fix lint for charts icon (#27243)

adiprerepa committed:
Add WorkloadGroup to Fuzz Test (#27231)

* add workloadgroup to fuzz

* add workloadgroup to fuzz

* pull latest istio/api and make gen

* populate HealhCheckMethod interface

* remove schema.test + readinessprobe

Commit:
Hide instead of remove (#27119)

howardjohn committed:
tf: cleanup docker deadcode (#27196)

* cleanup dead docker code

* Cleanup go.mod

istio-testing committed:
Automator: update proxy@master in istio/istio@master (#27236)

istio-testing committed:
Automator: update common-files@master in istio/istio@master (#27245)

howardjohn committed:
Remove duplicated variable setup (#27246)

istio-testing committed:
Automator: update istio/api@master dependency in istio/istio@master (#27215)

rshriram committed:
agent dns: optimize cname resolution (#27247)

* optimize cname resolution

Signed-off-by: Shriram Rajagopalan <rshriram@tetrate.io>

* lint

Signed-off-by: Shriram Rajagopalan <rshriram@tetrate.io>

* fix

Signed-off-by: Shriram Rajagopalan <rshriram@tetrate.io>

istio-testing committed:
Automator: update proxy@master in istio/istio@master (#27254)

howardjohn committed:
Fix headless auto protocol services (#27021)

* Fix headless auto protocol services

Fixes https://github.com/istio/istio/issues/27018

* Add tests

rlenglet committed:
sort the output of istioctl profile list (#27262)

adiprerepa committed:
add http health checks

adiprerepa committed:
com

istio-testing commented:
@adiprerepa: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integ-multicluster-k8s-tests_istio | 4f86e075e2399d5281b786fe196b3fc798cde14f | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/integ-multicluster-k8s-tests_istio/10230) | `/test integ-multicluster-k8s-tests_istio`
release-notes_istio | 5a67a6e916e9409012f0dce9c56b6f8a2f6c88f4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/27160/release-notes_istio/3266) | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->
